Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Linq;
+using Content.Client.Station;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.Administration.UI.Tabs.ObjectsTab;
+
+[GenerateTypedNameReferences]
+public sealed partial class ObjectsTab : Control
+{
+    [Dependency] private readonly EntityManager _entityManager = default!;
+
+    private readonly List<ObjectsTabEntry> _objects = new();
+    private List<ObjectsTabSelection> _selections = new();
+
+    public event Action<BaseButton.ButtonEventArgs>? OnEntryPressed;
+
+    public ObjectsTab()
+    {
+        RobustXamlLoader.Load(this);
+        IoCManager.InjectDependencies(this);
+
+        ObjectTypeOptions.OnItemSelected += ev =>
+        {
+            ObjectTypeOptions.SelectId(ev.Id);
+            RefreshObjectList(_selections[ev.Id]);
+        };
+
+        foreach (var type in Enum.GetValues(typeof(ObjectsTabSelection)))
+        {
+            _selections.Add((ObjectsTabSelection)type!);
+            ObjectTypeOptions.AddItem(Enum.GetName((ObjectsTabSelection)type!)!);
+        }
+
+        RefreshObjectList(_selections[ObjectTypeOptions.SelectedId]);
+    }
+
+    private void RefreshObjectList(ObjectsTabSelection selection)
+    {
+        var entities = selection switch
+        {
+            ObjectsTabSelection.Stations => _entityManager.EntitySysManager.GetEntitySystem<StationSystem>().Stations.ToList(),
```

Comment:
```suggestion
            ObjectsTabSelection.Stations => _entityManager.EntitySysManager.GetEntitySystem<StationSystem>(true).Stations.ToList(),
```

Given grids and maps also have their paused ones returned?

Reply:
I can tell you were tired.